### PR TITLE
Distinguish between the library/client/RM

### PR DESCRIPTION
**Pull request by jjhursey**
Distinguish between the library/client/RM responsibilities and the role of the standard.

### CODE REVIEW

File: Chap_Introduction.tex [hunk 1]
```diff
@@ -89,6 +98,25 @@ \subsection{What is \emph{not} defined in the standard?}
 
 The flip side of that statement is also true: Users who find that their current vendor does not support a function or attribute they require may raise that concern to their vendor and request that the implementation be expanded. Alternatively, users may wish to utilize the \ac{PRRTE} as a ``shim'' between their application and the host environment as it might provide the desired support until the vendor can respond. Finally, in the extreme, one can exploit the portability of PMIx-based applications to change vendors.
 
+%%%%%%%%%%%
+\subsection{General Guidance for PMIx Users and Implementors}
+
+The \ac{PMIx} Standard defines the behavior of the \acf{PRL}.
+A complete system harnessing the \ac{PMIx} interface requires an agreement between the \ac{PMIx} client, be it a tool or library, and the \ac{PMIx}-enabled \ac{RM}.
+The \ac{RPL} acts as an intermediary between these two entities providing a standard API for the exchange of requests and responses.
+The degree to which the \ac{PMIx} client and the \ac{PMIx}-enabled \ac{RM} may interact needs to be defines by those developer communities.
```
Comment (artpol84) [inline Chap_Introduction.tex]: `to be defines` - > `to be defined` ?

File: Chap_Introduction.tex [hunk 2]
```diff
@@ -52,24 +52,33 @@ \section{Charter}
...
+The \ac{PMIx} Standard defines and describes the interface developed by the \acf{PRL}.
+Much of this document is specific to the \acf{PRL}'s design and implementation.
+Specifically the standard describes the functionality provided by the \ac{PRL}, and what the \ac{PRL} requires of the clients and \acfp{RM} that use it's interface.
 
 %%%%%%%%%%%
 \subsection{Who should use the standard?}
 
-\ldots
+The \ac{PMIx} Standard informs PMIx clients and \acp{RM} of the syntax and semantics of the \ac{PMIx} APIs.
+
```
[the remark this answers is not in the document]
Comment (jjhursey) [reply Chap_Introduction.tex]: This is a fancy trick of the acronym package
`\ac{RM}` is singular `resource manager (RM)`
`\acp{RM}` is plural `resource managers (RMs)`

File: Chap_Introduction.tex [hunk 3]
```diff
@@ -52,24 +52,33 @@ \section{Charter}
...
+\ac{PMIx} clients (e.g., tools, \ac{MPE} libraries) can use this standard to understand the set of attributes provided by various APIs of the \ac{RPL} and their intended behavior.
+Additional information about the rationale for a selection of interfaces/attributes is also provided.
+
```
[the remark this answers is not in the document]
Comment (jjhursey) [reply Chap_Introduction.tex]: fixed

File: Chap_Introduction.tex [hunk 4]
```diff
@@ -52,24 +52,33 @@ \section{Charter}
...
+\ac{PMIx} \acp{RM} can use this standard to understand the expected behavior required of them when they support various interfaces/attributes.
```
[the remark this answers is not in the document]
Comment (jjhursey) [reply Chap_Introduction.tex]: not here, RMs is plural so we are referencing them as we do with PMIx clients above

File: Chap_Introduction.tex [hunk 5]
```diff
@@ -52,24 +52,33 @@ \section{Charter}
...
+Additional, optional features and suggestions on behavior are also included in the discussion to help guide \ac{RM} design and implementation.
 
 %%%%%%%%%%%
 \subsection{What is defined in the standard?}
 
-\ldots
+The \ac{PMIx} Standard defines the and describes the interface developed by the \acf{PRL}.
+It defines the set of attributes that the \ac{PRL} supports, the set of attributes that are required of a \ac{RM} to support for a given interface, and the set of optional attributes that an \ac{RM} may choose to support for a given interface.
```
[the remark this answers is not in the document]
Comment (jjhursey) [reply Chap_Introduction.tex]: added punctuation to clarify usage

**Pull request merged**